src/main.rs

```diff
@@ -75,6 +75,29 @@ fn get_issue_tree_init_hashes(repo: &Repository, _: &clap::ArgMatches) -> i32 {
 }
 
 
+/// create-message subcommand implementation
+///
+fn create_message(repo: &Repository, matches: &clap::ArgMatches) -> i32 {
+    let issue = matches.value_of("issue")
+                       .and_then(|i| Oid::from_str(i).ok()); //TODO: this hides errors
+    let sig = try_or_1!(repo.signature());
+
+    let parents : Vec<Commit> = match matches.values_of("parents") {
+        Some(values) => values.filter_map(|parent| Oid::from_str(parent).ok()) //TODO: this hides errors
+                              .filter_map(|oid| repo.find_commit(oid).ok()) //TODO: this hides errors
+                              .collect(),
+        _            => Vec::new(),
+    };
+    let parent_refs : Vec<&Commit> = parents.iter().map(|command| command).collect();
```

Comment:
Why not merging this piece of code with the above `collect()`? I know we want `&[&Commit]` instead of `&[Commit]`, but it should be possible without this boilerplate line!